Cargo.toml

```diff
@@ -1,2 +1,25 @@
-[workspace]
-members = [ "libtest" ]
\ No newline at end of file
+[package]
+authors = ["The Rust Project Developers"]
+name = "libtest"
+version = "0.0.2"
+edition = "2018"
+description = "Rust's built in unit-test and micro-benchmarking framework"
+license = "MIT/Apache-2.0"
+documentation = "https://docs.rs/libterm"
+homepage = "https://github.com/rust-lang/libtest"
+repository = "https://github.com/rust-lang/libtest"
+readme = "README.md"
+
+[lib]
+name = "libtest"
+path = "src/lib.rs"
+crate-type = ["dylib", "rlib"]
```

Comment:
I think this can be removed because `test` itself in rust-lang/rust will be responsible for being a dylib